project.clj

```diff
@@ -3,13 +3,13 @@
             :url "https://opensource.org/licenses/MIT"}
   :source-paths ["src"]
   :description "A hello world application for electron"
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/clojurescript "1.9.542"]
-                 [figwheel "0.5.10"]
+  :dependencies [[org.clojure/clojure "1.9.0"]
+                 [org.clojure/clojurescript "1.10.238"]
```

Comment:
Try to downgrade cljs compiler back to `1.9.542` to see if this fixes the build.

Reply:
It works with `1.9.542`, I'm gonna try to find the root cause. It would be nice to get it working with `1.10.x` since it improved build time significantly.
If it's a regression of `1.10.x`, I'll downgrade cljs.